watertap/core/membrane_channel_base.py

```diff
@@ -89,8 +99,7 @@ class FrictionFactor(Enum):
     spiral_wound: Darcy's friction factor correlation by Schock & Miquel, 1987
```

Comment:
Can replace this doc string to explain how default_by_module_type will revert to either flat_sheet correlation by G&H or spiral_wound correlation by S&M

Reply:
Done!